types/node/index.d.ts

```diff
@@ -238,6 +238,10 @@ declare var Buffer: {
      * @param arrayBuffer The .buffer property of a TypedArray or a new ArrayBuffer()
      */
     from(arrayBuffer: ArrayBuffer, byteOffset?: number, length?: number): Buffer;
+    /**
+     * Copies the passed buffer data onto a new Buffer instance.
+     */
+    from(buffer: Uint8Array): Buffer;
```

Comment:
Why not add `| Uint8Array` to the `from(data: any[]...)` function below instead adding a new overload?

Reply:
I did it because the documentation is different on https://nodejs.org/api/buffer.html. The overload with a `Uint8Array` copies the data while the other overloads create a new Buffer using the passed data. 

Should I add it to the overload even though the semantic and documentation are different?

Reply:
My comment was based on the changes done in #24966 where `Buffer.from()` overloads have been combined to fix some issue related to how typescript does overload resolution.
My feeling is that we should avoid overloads because of this. As long as documentation is up to date it should be fine.